go/cmd/remotetool/main.go

```diff
@@ -0,0 +1,72 @@
+// Main package for the remotetool binary.
```

Comment:
Can you add documentation for an example use of the tool?

Reply:
Done.